### DISCUSSION TITLE
[ Bun.serve ] how to turn on Version HTTP/2 ?

### DISCUSSION THREAD

**Discussion by truongan07** [oven-sh/bun#11289] [Q&A]
- On Nginx : [http2 on] in configuration and ...

![image](https://github.com/oven-sh/bun/assets/141234261/e74cfdcf-7942-47db-853f-addaa7897163)

- On Bun : what is the param ?
 
![image](https://github.com/oven-sh/bun/assets/141234261/784741c4-6137-4f8c-ba77-343821a27184)

- I want to run TLS of Bun.serve on 443 port without Nginx.

**Comment by re-thc** [comment]
It's not supported yet. See https://github.com/oven-sh/bun/issues/8823

**Comment by Jarred-Sumner** [comment]
I wish it were as simple as a flag to turn on

**Reply by truongan07** [reply]
Hi my friend, You and Bun team are talented persons. I think Bun team can do complex become simple
